.cargo/config.toml

```diff
@@ -1,19 +1,45 @@
-#[target.xtensa-esp32s3-none-elf]
+[alias]
+build-esp32   = "build --release --target xtensa-esp32-none-elf --features esp32"
+build-esp32c2 = "build --release --target riscv32imc-unknown-none-elf --features esp32c2"
+build-esp32c3 = "build --release --target riscv32imc-unknown-none-elf --features esp32c3"
+#build-esp32c5 = "build --release --target riscv32imac-unknown-none-elf --features esp32c5"
+build-esp32c6 = "build --release --target riscv32imac-unknown-none-elf --features esp32c6"
+build-esp32s2 = "build --profile esp32s2 --target xtensa-esp32s2-none-elf --features esp32s2"
+build-esp32s3 = "build --release --target xtensa-esp32s3-none-elf --features esp32s3"
+#build-esp32p4 = "build --release --target riscv32imac-unknown-none-elf --features esp32p4"
+
+run-esp32   = "run --release --target xtensa-esp32-none-elf --features esp32"
+run-esp32c2 = "run --release --target riscv32imc-unknown-none-elf --features esp32c2"
+run-esp32c3 = "run --release --target riscv32imc-unknown-none-elf --features esp32c3"
+# Available but not supported by esp-hal (yet).
```

Comment:
Regarding C5 support: https://github.com/esp-rs/esp-hal/issues/3454

Reply:
Thanks Sergio! Seems that the [SVD still has to come out](https://github.com/espressif/svd/tree/main/svd) (and thus no PAC generated yet), let's leave it like that for now, it's just too new ATM.